src/Pomelo.EntityFrameworkCore.MySql/Storage/Internal/MySqlDatabaseCreator.cs

```diff
@@ -232,10 +233,10 @@ private IReadOnlyList<MigrationCommand> CreateDropCommands()
         }
 
         // Clear connection pools in case there are active connections that are pooled
-        private static void ClearAllPools() => MySqlConnection.ClearAllPools();
+        private static void ClearAllPools() => MySqlHelper.ClearConnectionPools();
 
         // Clear connection pool for the database connection since after the 'create database' call, a previously
         // invalid connection may now be valid.
-        private void ClearPool() => MySqlConnection.ClearPool((MySqlConnection)_connection.DbConnection);
+        private void ClearPool() => _connection?.Close();
```

Comment:
MySqlConnector needs to implement the API you need here: https://github.com/bgrainger/MySqlConnector/issues/46